package.json

```diff
@@ -101,6 +101,8 @@
     "redis": ">=3.1.2",
     "hosted-git-info": ">=3.0.8",
     "lodash": ">=4.17.21",
-    "ws": ">=7.4.6"
+    "ws": ">=7.4.6",
+    "glob-parent": ">=5.1.2",
+    "normalize-url": ">=4.5.1 <5.0.0"
```

Comment:
do we need to restrict the upper version?

Btw there's a tool https://www.npmjs.com/package/yarn-audit-fix that patches the vulnerable dependencies right in yarn.lock file - doesn't always solve everything, but when it does, at least you don't have to "hack" package.json with resolutions which stay and may eventually result in issues related to the package version forced being too old

Reply:
I removed the upper bound